lib/rdl/constraint.rb

```diff
@@ -1,4 +1,5 @@
 require 'csv'
+#require 'method_source'
```

Comment:
@jeffrey-s-foster This was required to prevent the system from crashing when it read from `comment`, but I noticed after I'd merged `infer` back into this branch, that all of that code was commented out. Still, I thought I might leave it here if we ever wanted to record the comments again.

Reply:
Suggest moving this to `boot.rb` rather than here